src/lib/gadgets/gadgets.unit-test.ts

```diff
@@ -116,3 +117,115 @@ test(
     });
   }
 );
+
+// --------------------------
+// Shift Gates
+// --------------------------
+
+function testShift(
+  field: Field,
+  bits: number,
+  mode: 'left' | 'right',
+  result: Field
+) {
+  Provable.runAndCheck(() => {
+    let w = Provable.witness(Field, () => field);
+    let r = Provable.witness(Field, () => result);
+    let output = Provable.if(
```

Comment:
nit: we can just use a normal JS `if` statement here

Reply:
Fixed in https://github.com/o1-labs/o1js/pull/1194/commits/b7e427c24a0396556450836a18302814a511ee74